internal/events/eventhandlers/patch_anchor_platform_transaction_completion_event_handler.go

```diff
@@ -0,0 +1,91 @@
+package eventhandlers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/anchorplatform"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/crashtracker"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/data"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/events"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/events/schemas"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/services"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/utils"
+	"github.com/stellar/stellar-disbursement-platform-backend/stellar-multitenant/pkg/tenant"
+)
+
+type PatchAnchorPlatformTransactionCompletionEventHandlerOptions struct {
+	DBConnectionPool   db.DBConnectionPool
+	APapiSvc           anchorplatform.AnchorPlatformAPIServiceInterface
+	CrashTrackerClient crashtracker.CrashTrackerClient
+}
+
+type PatchAnchorPlatformTransactionCompletionEventHandler struct {
+	tenantManager      tenant.ManagerInterface
+	service            services.PatchAnchorPlatformTransactionCompletionServiceInterface
+	crashTrackerClient crashtracker.CrashTrackerClient
+}
+
+var _ events.EventHandler = new(PatchAnchorPlatformTransactionCompletionEventHandler)
+
+func NewPatchAnchorPlatformTransactionCompletionEventHandler(options PatchAnchorPlatformTransactionCompletionEventHandlerOptions) *PatchAnchorPlatformTransactionCompletionEventHandler {
+	s, err := services.NewPatchAnchorPlatformTransactionCompletionService(options.APapiSvc, nil)
+	if err != nil {
+		log.Fatalf("error instantiating service: %s", err.Error())
+	}
+
+	return &PatchAnchorPlatformTransactionCompletionEventHandler{
+		tenantManager:      tenant.NewManager(tenant.WithDatabase(options.DBConnectionPool)),
+		service:            s,
+		crashTrackerClient: options.CrashTrackerClient,
+	}
+}
+
+func (h *PatchAnchorPlatformTransactionCompletionEventHandler) Name() string {
+	return "PatchAnchorPlatformTransactionCompletionEventHandler"
+}
+
+func (h *PatchAnchorPlatformTransactionCompletionEventHandler) CanHandleMessage(ctx context.Context, message *events.Message) bool {
+	return message.Topic == events.PatchAnchorPlatformTransactionCompletionTopic
```

Comment:
I think it's possible to use one topic for this handler and the one that synchronizes TSS txs back to the SDP. 
Meaning having a `payments.payment_completed` topic that both event handlers can read from